src/routes/services.ts

```diff
@@ -67,10 +71,11 @@ export const vpn = {
 			});
 	},
 
-	clientDisconnect: (req: Request, res: Response) => {
+	clientDisconnect: (req: Request, res: Response): Promise<void> => {
```

Comment:
The real response type is `void | Promise<void>` - which is why I think you've done the weird type casting below